user-guide/mkdocs.yml

```diff
@@ -1,39 +1,33 @@
+INHERIT: ./mkdocs.base.yml
+
 site_name: DesignSafe User Guide
 site_description: DesignSafe User Guide
 site_url: https://designsafe-ci.org/user-guide/
-plugins:
-  - search
-  - include-markdown
+# NOTE: Unrecognized by MkDocs, but recognized by ReadTheDocs theme
+site_favicon: https://www.designsafe-ci.org/favicon.ico
 
 theme:
     name: readthedocs
-    navigation_depth: 3
-    collapse_navigation: false
-    custom_dir: themes/ds-readthedocs
-    highlightjs: true
+    logo: https://www.designsafe-ci.org/static/images/org_logos/Horizontal-DS.b190a072b800.jpg
+    cms_url: https://designsafe-ci.org/ # TACC Feature
+    cms_name: designsafe-ci.org # TACC Feature
     features:
       - navigation.top
-    ds_navigation_redirects:
+    nav_redirects: # TACC Feature
       - before: '#data-depotcuration-office-hours'
         after: https://www.designsafe-ci.org/facilities/virtual-office-hours/
-    analytics:
-      gtag: G-D96RT1T24K
 
 extra_css:
-    - css/ds-docs.css
-
-extra_javascript:
-    - js/changeNavMarkup.js
-    # some JavaScript is included as modules in `./themes/ds-readthedocs/`
+  # TACC/TACC-Docs:/mkdocs.base.yml
+  - css/core/bootstrap.css
+  - css/core/core-styles.css
+  - css/core/tacc-docs.css
+  # DesignSafe/DS-User-Guide
+  - css/ds-docs.css
 
-markdown_extensions:
-  - footnotes
-  - def_list
-  - admonition
-  - attr_list
-  - abbr
-  - toc:
-      permalink: True
```

Comment:
The `toc` plugin is enabled by MkDocs by default ([source a](https://github.com/TACC/TACC-Docs/blob/6b1dd1b/mkdocs.yml#L118-L121), [source b](https://github.com/mkdocs/mkdocs/blob/1.4.2/docs/user-guide/configuration.md#markdown_extensions)).

Reply:
No, the default is `False` ([ref.](https://python-markdown.github.io/extensions/toc/#usage)).

But irrelevant, because DS-User-Guide does **not** use [`[TOC]`](https://python-markdown.github.io/extensions/toc/#syntax) ([ref.](https://github.com/search?q=repo%3ADesignSafe-CI%2FDS-User-Guide+%22%5BTOC%5D%22&type=code)).

I tested on [TACC/TACC-Docs which has one instance](https://github.com/TACC/TACC-Docs/blob/2178fe4/docs/code-examples/index.md?plain=1#L3).

| `True` | `False` | default |
| - | - | - |
| <img width="900" alt="True" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/312184c3-ef5d-45e3-ab53-806f69c6a1d4"> | <img width="900" alt="False" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/b8be1db6-ab31-4ad4-b584-89060f528313"> | <img width="900" alt="default" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/a5cf674d-33f5-48ef-b012-0563020d0ad6"> |